uktrade/data-hub-frontend
# Replace policy feedback with business intelligence.

## Description of change

This replaces "policy feedback" with "business intelligence" in the interaction filter and interaction badges.

## Test instructions

Go to interaction collection page and see "business intelligence" in place of "policy feedback".

## Screenshots
### Before

![bi-before](https://user-images.githubusercontent.com/5889630/103997074-8c8c1d80-5192-11eb-9c3d-5e3fefd07559.png)

### After

![bi-after](https://user-images.githubusercontent.com/5889630/103997092-901fa480-5192-11eb-877e-211a12277ac5.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
